modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -279,7 +279,7 @@ spec:
     node-role.kubernetes.io/worker-rt: ""
 ----
 
-For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html-single/nodes/index#nodes-scheduler-node-selectors[Placing pods on specific nodes using node selectors].
+For more information, see _Placing pods on specific nodes using node selectors_ in the _Additional resources_ section.
```

Comment:
Per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations)
```suggestion
For more information, see _Placing pods on specific nodes using node selectors_.
```